tests/test_copilot_cli.py

```diff
@@ -0,0 +1,72 @@
+from pathlib import Path
+
+from click.testing import CliRunner
+import pytest
+import yaml
+
+from commands.copilot_cli import copilot as cli, WAF_ACL_ARN_KEY
+
+
+class TestApplyWAF:
+
+    def test_require_copilot_dir(self, fakefs):
```

Comment:
Could we use more meanigful names for the test methods so that it's clearer what the intent of the test is and thereby easier to be sure it's verifying the correct thing?

E.g.
```suggestion
    def test_apply_waf_with_no_copiliot_directory_throws_error(self, fakefs):
```

Reply:
The test are fairly self explanatory, but I'm happy make them even more descriptive.

The `apply_waf` bit is evident in the containing class name.

```
class TestApplyWaf:
```

To make it clearer I'll rename it to 

```
class TestApplyWafCommand:
```

Maybe a better name for this method then would be `test_exit_if_no_copilot_directory()` ?